paddle/go/pserver/lib/client/main.go

```diff
@@ -0,0 +1,239 @@
+package main
```

Comment:
Is this `main.go` program going to compile to a shared library like `.so`, so that trainers which are written in python can call this lib? If so, we should add a CPython wrapper to call this CGo program, and the `paddle/go/pserver/lib/client/test/main.c` should be a python example. Current `paddle/go/pserver/lib/client/test/main.c` can be moved to the python wrapper functions.